Cargo.toml

```diff
@@ -1,22 +1,22 @@
 [package]
-name          = "librpm"
-version       = "0.1.1"
-description   = "RPM Package Manager library binding with an idiomatic Rust API"
-license       = "MPL-2.0"
-authors       = ["Tony Arcieri <tony@iqlusion.io>"]
-categories    = ["api-bindings", "os", "parsing"]
-keywords      = ["rpm", "linux", "redhat", "fedora", "centos"]
-readme        = "README.md"
-homepage      = "https://github.com/rpm-software-management/librpm.rs/"
+name = "librpm"
+version = "0.2.0"
```

Comment:
If we're cutting a breaking release, there are some other changes to consider (also the formatting changes make it a bit unclear the version is being bumped here), like bumping the edition.

Perhaps you could just bump to `0.2.0-pre` for now?

Reply:
Was thinking of doing that in a different PR but im fine with just bumping the edition here. The autoformatting was done automatically.

Reply:
Smaller, easier-to-review PRs would be preferable. There's already a lot going on in this PR.

Reply:
That what I suggested, but at this point this repo is in a state where a single change incurs other widespread changes (formatting, edition version, CI, working tests, etc).
Please let me know how would you like to proceed with this PR. As I see it, nearly all the changes done here are mandatory, and taking this PR in would get this repo to be 2023 ready :)

Reply:
Fixing CI would probably be a first step. See also #43

Reply:
Please see https://github.com/rpm-software-management/librpm.rs/pull/46